topics/json-schema/index.md

```diff
@@ -0,0 +1,13 @@
+---
+created_by: Kris Zyp
```

Comment:
Nit: Is this person intentional?

Reply:
Thanks for the comment @egekorkan. Kris was the creator of the Spec, however he is not currently involved in the project. Your comment made me think and I will add here the  complete list of authors of all the versions of the Spec. Thanks!!